robottests/translation.txt

```diff
@@ -29,6 +29,7 @@ Can switch to other language  [Arguments]  ${button}  ${language}
 Enter Translation
     Open Browser  ${BASE_URL}  ${BROWSER}
     Click Link  Translations
+    Set Browser Implicit Wait     0.2 seconds
```

Comment:
It's the only way I found to make browser sleep for a bit while the image gets loaded. This adds ~1 sec to overall tests.